src/libs/ReportActionsUtils.js

```diff
@@ -425,6 +425,10 @@ function getLinkedTransactionID(reportID, reportActionID) {
     return reportAction.originalMessage.IOUTransactionID;
 }
 
+function getReportAction(reportID, reportActionID) {
```

Comment:
Sorry for any confusion, but we do still need a JS doc for the parameters and return value. We don't need a description however.

Reply:
Resolved. And sorry on the test changes missed it due to my search exclude of VS code. 😓